app/controllers/discourse_assign/assign_controller.rb

```diff
@@ -174,6 +174,50 @@ def group_members
              }
     end
 
+    USER_MENU_LIMIT = 20
```

Comment:
Are we able to pull this constant from core instead of defining it here? It'll at least help to ensure any changes in core is automatically reflected in the plugin.

Reply:
done!